src/catalog/src/builtin.rs

```diff
@@ -4191,6 +4263,106 @@ WHERE worker_id = 0",
     sensitivity: DataSensitivity::Public,
 };
 
+pub const MZ_ARRANGEMENT_BATCHER_RECORDS_PER_WORKER: BuiltinView = BuiltinView {
+    name: "mz_arrangement_batcher_records_per_worker",
+    schema: MZ_INTERNAL_SCHEMA,
+    column_defs: None,
+    sql: "
+SELECT
+    operator_id,
+    worker_id,
+    pg_catalog.count(*) AS count
+FROM mz_internal.mz_arrangement_batcher_records_raw
+GROUP BY operator_id, worker_id",
+    sensitivity: DataSensitivity::Public,
+};
+
+pub const MZ_ARRANGEMENT_BATCHER_RECORDS: BuiltinView = BuiltinView {
+    name: "mz_arrangement_batcher_records",
+    schema: MZ_INTERNAL_SCHEMA,
+    column_defs: None,
+    sql: "
+SELECT operator_id, sum(count)
+FROM mz_internal.mz_arrangement_batcher_records_per_worker
+GROUP BY operator_id",
+    sensitivity: DataSensitivity::Public,
+};
+
+pub const MZ_ARRANGEMENT_BATCHER_SIZE_PER_WORKER: BuiltinView = BuiltinView {
+    name: "mz_arrangement_batcher_size_per_worker",
+    schema: MZ_INTERNAL_SCHEMA,
+    column_defs: None,
+    sql: "
+SELECT
+    operator_id,
+    worker_id,
+    pg_catalog.count(*) AS count
+FROM mz_internal.mz_arrangement_batcher_size_raw
+GROUP BY operator_id, worker_id",
+    sensitivity: DataSensitivity::Public,
+};
+
+pub const MZ_ARRANGEMENT_BATCHER_SIZE: BuiltinView = BuiltinView {
+    name: "mz_arrangement_batcher_size",
```

Comment:
This should be plural `mz_arrangement_batcher_sizes` (as per https://github.com/MaterializeInc/materialize/blob/b290d40f1bd923ce5d525526e10e1cf9853901ff/doc/developer/style.md?plain=1#L140-L142)

Reply:
Also removed, but thanks for the pointer!